### PR TITLE
get rid of filamat-lite, which was poorly maintained

### PR DESCRIPTION
**Pull request by pixelflinger**

### CODE REVIEW

**Comment by romainguy** [comment]
You'll need to update README.md as well as it mentions it.

**Review by romainguy — CHANGES_REQUESTED** [review]

**Review by bejado — APPROVED** [review]

File: android/filamat-android/build.gradle [hunk 1]
```diff
@@ -6,27 +6,13 @@ android {
         full {
             dimension "functionality"
         }
-
```
[the remark this answers is not in the document]
Comment (pixelflinger) [reply android/filamat-android/build.gradle:9]: done

File: android/samples/sample-material-builder/build.gradle [hunk 2]
```diff
@@ -32,9 +32,6 @@ android {
         targetSdkVersion versions.targetSdk
     }
 
-    // The filamat library has two variants: full and lite. Here we default to the "full" variant.
-    // Replace "full" with "lite" to use the filamat-lite variant, which has a smaller binary size
-    // but comes with certain restrictions. See "Filamat Lite" in libs/filamat/README.md.
```
[the remark this answers is not in the document]
Comment (pixelflinger) [reply android/samples/sample-material-builder/build.gradle:37]: done

File: android/filamat-android/build.gradle [hunk 3]
```diff
@@ -6,27 +6,13 @@ android {
...
-            externalNativeBuild {
-                cmake {
-                    arguments.add("-DFILAMAT_LITE=ON")
-                }
-            }
-        }
     }
 
     publishing {
         singleVariant("fullRelease") {
```
[the remark this answers is not in the document]
Comment (pixelflinger) [reply android/filamat-android/build.gradle]: done

**Review by romainguy — APPROVED** [review]

**Pull request merged**